model/pytorch/tests/test_pytorchnet.py

```diff
@@ -183,6 +184,57 @@ async def test_02_predict(self, predictdir):
         self.assertIn(results["value"], self.model.config.classifications)
         self.assertTrue(results["confidence"])
 
+    async def test_03_network_creation_from_dict(self):
+        model1 = {
+            "linear1": {
+                "layer_type": "Linear",
+                "in_features": 4096,
+                "out_features": 256,
+            },
+            "relu": {"layer_type": "ReLU"},
+            "linear_2": {
+                "layer_type": "Linear",
+                "in_features": "",
+                "out_features": 17,
+            },
+        }
+        model2 = {
+            "conv1": {
+                "layer_type": "Conv2d",
+                "in_channels": 3,
+                "out_channels": 32,
+                "kernel_size": 3,
+            },
+            "relu1": {"layer_type": "ReLU"},
+            "conv2": {
+                "layer_type": "Conv2d",
+                "in_channels": None,
+                "out_channels": 64,
+                "kernel_size": 3,
+            },
+            "relu2": {"layer_type": "ReLU"},
+            "conv3": {
+                "layer_type": "Conv2d",
+                "in_channels": "",
```

Comment:
We should be able to make it such that we don't need to mention it at all
```suggestion
```

Reply:
Yes the function which I did actually handles it , as this input is valid, and the `conv3` layer will be taking its value from the  above `conv2` layers